setup.py

```diff
@@ -98,6 +98,18 @@
     DPCPP_LIBS = []
 DAAL_DEFAULT_TYPE = 'double'
 
+
+#Level Zero workaround for oneDAL Beta06
+from generator.parse import parse_version
+
+header_path = daal_root + '/include/services/library_version_info.h'
```

Comment:
It is better to use appropriate functions like os.path.join(daal_root, 'include', 'services', 'library_version_info.h')

Reply:
done